site/docs/transform/window.md

```diff
@@ -69,52 +69,171 @@ To replace any of the examples from total to `mean`, the operation just needs to
+                "type": "nominal",
+                "scale": { "rangeStep": 12 },
+                "axis": { "title": "" }
+            },
+            "y": {
+                "field": "PercentOfTotal",
+                "type": "quantitative",
+                "axis": { "title": "% of total Time", "grid": false }
+            }
+        }
     }],
-    "frame": [null, null]
-  },
-  {
-    "calculate": "datum.x/datum.total",
-    "as": "Percent of Total"
-  }
+    "config": {
+        "view": { "stroke": "transparent" },
+        "axis": { "domainWidth": 1 }
+    }
+}
 ```
 
 
-### Difference from Total
+### Difference from Mean
+
+<div class="vl-example" data-name="window_transform_movie_mean_difference"></div>
 
 ```json
-  {
-    "window": [{
-        "op": "sum",
-        "field": "x",
-        "as": "total"
-    }],
-    "frame": [null, null]
-  },
-  {
-    "calculate": "datum.total - datum.x",
-    "as": "Difference from total"
-  }
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+    "description": "Bar graph showing how each movie is different from the average movie rating",
+    "width": 300,
+    "height": 50,
+    "data": { "url": "data/movies.json" },
+    "layer": [{
+        "transform": [{
+                "window": [{
+                    "op": "mean",
+                    "field": "IMDB_Rating",
+                    "as": "AverageRating"
+                }],
+                "frame": [null, null]
+            },
+            {
+                "calculate": "datum.IMDB_Rating - datum.AverageRating",
+                "as": "DifferenceFromMean"
+            },
+            {
+                "filter": "datum.DifferenceFromMean > 2.5"
+            }
+        ],
+        "mark": { "type": "bar", "clip": true },
+        "encoding": {
+            "x": {
+                "field": "Title",
+                "type": "ordinal"
+            },
+            "y": {
+                "field": "IMDB_Rating",
+                "type": "quantitative",
+                "scale": { "domain": [7, 10] },
+                "axis": { "title": "IMDB rating" }
+            }
+        }
+    }]
+}
 ```
 
-### Percent Difference from Total
+### Percent Difference from mean
+
+The example above, could be adjusted to be the percent difference instead of the absolute difference, by adjusting the window transform to be in the format below.
 
 ```json
   {
     "window": [{
-        "op": "sum",
+        "op": "mean",
         "field": "x",
-        "as": "total"
+        "as": "average"
     }],
     "frame": [null, null]
   },
   {
-    "calculate": "(datum.total - datum.x)/dataum.total",
-    "as": "Percent difference from total"
+    "calculate": "(datum.total - datum.x)/dataum.average * 100",
+    "as": "Percent difference from average"
   }
-```
\ No newline at end of file
+```
+
+## Other examples
+
+Here is an example where the window_transform can be used to get the average rating for a film during different years.
```

Comment:
Why is this "other example"?   This is another example of difference from mean.   Seems like the  DIV tag is also pointing to the same example as above.  

I'd say group this in the `### Difference from mean` section.  

In fact, I'd say that remove the "Movie 1, Movie 2, Movie 3, Movie 4" example as it's as interesting as the real one. 

If your point is to show that you can group by year, you can create multiple variations of the  same example like:

- [ ] no group by and use the value to filter
- [ ] group by year and use the value to filter
- [ ] no group by, but visualize residual  (rating - mean_rating) instead of just filtering. 

This way users can easily see the changes without having to totally learn about a totally different example.